src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Constants.cs

```diff
@@ -24,4 +24,9 @@ public class PolicyConstants
         public const int MinPolicyNameLength = 3;
         public const int MaxPolicyNameLength = 150;                
     }
+
+    public class ContainerConstansts
+    {
+        public const string SqlContainerNamePrefix = "AzureSqlContainer;";
```

Comment:
This prefix seems to be ContainerType + ";". Isn't it better to use the ContainerType itself rather than create a new constant for better code manageability?